docs/api-testing/apifctl-cicd-integration.md

```diff
@@ -345,42 +319,106 @@ Specifies your tunnel ID for running tests using Sauce Connect Proxy.
 ### `-p <local path to file>`
 <p><small>| STRING |</small></p>
 
-**For the [`vault-update`](#vault-update) command**: This is the local path to the folder containing your Vault file. Example: `-p /tests/vault/vault.json`.
+For the **[`exec`](#exec)** and **[`upload`](#upload)** commands: This is the local path to the folder containing both your unit.xml and input.xml test files. In the below example, `product_update` is the folder containing unit.xml and input.xml.
+
+```bash
+-p /tests/product_update
+```
+
+For the **[`vault-update`](#vault-update)** command: This is the local path to the folder containing your Vault file.
+
+```bash
+-p /tests/vault/vault.json
+```
+
+---
+### `-f <from time>`
+<p><small>| STRING |</small></p>
+
+Identifies the start date of the events you want to see. For use with **[`metrics`](#metrics)** and **[`events`](#events)** commands.
 
-**For the [`exec`](#exec) or [`upload`](#upload) command**: This is the local path to the folder containing your test files (both unit.xml and input.xml). Example: `-p /tests/product_update` (where product_update is the folder containing unit.xml and input.xml).
+```bash
+-f 2021-10-21T14:00
+```
+
+---
+### `-t <to time>`
+<p><small>| STRING |</small></p>
+
+Identifies the end date of the events you want to see. For use with **[`metrics`](#metrics)** and **[`events`](#events)** commands.
+
+```bash
+-t 2021-10-31T15:00
+```
+
+---
+### `-t <tag(s)>`
+<p><small>| STRING |</small></p>
+
+Adds a set of tags to the resulting event. Format as a comma-separated list of tags you want to assign to the test. For use with **[`exec`](#exec)** and **[`upload`](#upload)** commands.
+
+```bash
+-t product,production
+```
+
+---
+### `-tag <tag(s)>`
+<p><small>| STRING |</small></p>
+
+Adds a set of tags to the resulting event. Format as a comma-separated list of tags you want to assign to the test. For use with **[`run-tag`](#run-tag)** command only.
```

Comment:
-tag: the command (run-tag) will run all the tests in the project marked with that tag (e.g. - tag production: will run all the tests in the project that have "production" tag)